CHANGELOG.md

```diff
@@ -1,5 +1,24 @@
 # SAS Viya Monitoring for Kubernetes
 
+## Unreleased
+
+* **Overall**
+
+* **Metrics**
+  * [UPGRADE] - Kube-prometheus-stack has been upgraded from version 45.28.0 to 48.2.3
```

Comment:
Typo: the new version is (I believe) 48.3.2.

Reply:
Yep, that was a typo.  It'll be fixed in the next push.